package.json

```diff
@@ -45,43 +45,28 @@
   "author": "Mozilla",
   "license": "MPL-2.0",
   "dependencies": {
+    "autoprefixer": "^9.5.1",
     "@babel/core": "^7.4.4",
     "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
-    "autoprefixer": "^9.5.1",
     "babel-core": "^6.26.3",
     "babel-loader": "^8.0.6",
-    "browserslist": "^4.6.0",
-    "chokidar-cli": "^1.2.2",
     "classnames": "2.2.6",
-    "dotenv": "^8.0.0",
-    "eslint": "^5.16.0",
-    "eslint-config-prettier": "^4.3.0",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.13.0",
     "event-stream": "3.3.4",
     "html-entities": "^1.2.1",
     "js-cookie": "2.2.0",
     "mofo-bootstrap": "4.1.0",
     "mofo-style": "^2.4.0",
```

Comment:
Seems like we no longer uses `mofo-style`. I'll file a separate ticket to remove this.

Reply:
Filed: #3248